test/test_twobytwo_indexed.jl

```diff
@@ -498,5 +507,351 @@ solve!(m)
+
+set_value(endow[:l], get_value(endow[:l])*1.1)
+set_value(C[:ra], 157.)
+set_fixed!(C[:ra], true)
+solve!(m)
+# RA=157
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[l]†Y[x]")]) ≈ TwoxTwo_DemandIndPrice_Nest["x.L","RA=157"]#  52.4404424
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[k]†Y[x]")]) ≈ TwoxTwo_DemandIndPrice_Nest["x.K","RA=157"]#  47.6731295
+@test MPSGE.Complementarity.result_value(m._jump_model[:Y][:x]) ≈ TwoxTwo_DemandIndPrice_Nest["y.x","RA=157"]#  1.0488088
+@test MPSGE.Complementarity.result_value(m._jump_model[:Y][:y]) ≈ TwoxTwo_DemandIndPrice_Nest["y.y","RA=157"]#  1.0388601
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[l]†Y[y]")]) ≈ TwoxTwo_DemandIndPrice_Nest["y.L","RA=157"]#  21.1770571
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PF[k]†Y[y]")]) ≈ TwoxTwo_DemandIndPrice_Nest["y.K","RA=157"]#  28.8778051
+@test MPSGE.Complementarity.result_value(m._jump_model[:U]) ≈ TwoxTwo_DemandIndPrice_Nest["U._","RA=157"]#  1.0454821
+@test MPSGE.Complementarity.result_value(m._jump_model[:C][:ra]) ≈ TwoxTwo_DemandIndPrice_Nest["RA._","RA=157"]#  157
+@test MPSGE.Complementarity.result_value(m._jump_model[:PC][:x]) ≈ TwoxTwo_DemandIndPrice_Nest["PC.x","RA=157"]#  0.9979575
+@test MPSGE.Complementarity.result_value(m._jump_model[:PC][:y]) ≈ TwoxTwo_DemandIndPrice_Nest["PC.y","RA=157"]#  1.0075145
+@test MPSGE.Complementarity.result_value(m._jump_model[:PF][:l]) ≈ TwoxTwo_DemandIndPrice_Nest["PF.L","RA=157"]#  0.9515152
+@test MPSGE.Complementarity.result_value(m._jump_model[:PF][:k]) ≈ TwoxTwo_DemandIndPrice_Nest["PF.K","RA=157"]#  1.0466667
+@test MPSGE.Complementarity.result_value(m._jump_model[:PU][:l]) ≈ TwoxTwo_DemandIndPrice_Nest["PU.L","RA=157"]#  1.0011331
+@test MPSGE.Complementarity.result_value(m._jump_model[:PU][:k]) ≈ TwoxTwo_DemandIndPrice_Nest["PU.K","RA=157"]#  1.0011331
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC[x]†U")]) ≈ TwoxTwo_DemandIndPrice_Nest["DU.x","RA=157"]#  100.3182058
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC[y]†U")]) ≈ TwoxTwo_DemandIndPrice_Nest["DU.y","RA=157"]#  49.6833066
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PU[l]‡U")]) ≈ TwoxTwo_DemandIndPrice_Nest["SU.L","RA=157"]#  75
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PU[k]‡U")]) ≈ TwoxTwo_DemandIndPrice_Nest["SU.K","RA=157"]#  75
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC[x]‡Y[x]")]) ≈ TwoxTwo_DemandIndPrice_Nest["SY.x","RA=157"]#  100
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC[y]‡Y[y]")]) ≈ TwoxTwo_DemandIndPrice_Nest["SY.y","RA=157"]#  50
+@test MPSGE.Complementarity.result_value(m._jump_model[Symbol("PC→CUρC[ra]")]) ≈ TwoxTwo_DemandIndPrice_Nest["FDRA.L","RA=157"] + TwoxTwo_DemandIndPrice_Nest["FDRA.K","RA=157"] # 78.4111548 + 78.4111548
```

Comment:
We report a nested variable (e.g. PC→CUρC[ra] above) that doesn't have a clear analogue in the GAMS version (probably one could be created, potentially as a calculation). We don't report the 2 individual variables that do come out of GAMS with a straightforward report variable definition. Here the result for the variable we report = the sum of the 2 from GAMS, but that's not true when they're not equal later on. 
``` 
$REPORT
 V:FDRA(f)   D:PU(f)  DEMAND:RA```